### PR TITLE
Add IMultiCommitManager

### PR DESCRIPTION
**Pull request by Thealexbarney**

### CODE REVIEW

File: Ryujinx.HLE/HOS/Services/Fs/IFileSystemProxy.cs [hunk 1]
```diff
@@ -513,5 +513,18 @@ public ResultCode GetProgramIndexForAccessLog(ServiceCtx context)
 
             return ResultCode.Success;
         }
+
+        [Command(1200)]
+        public ResultCode OpenMultiCommitManager(ServiceCtx context)
```
Comment (AcK77) [inline Ryujinx.HLE/HOS/Services/Fs/IFileSystemProxy.cs]: Could be nice to get a symbol here.

File: Ryujinx.HLE/HOS/Services/Fs/IFileSystemProxy.cs [hunk 2]
```diff
@@ -513,5 +513,18 @@ public ResultCode GetProgramIndexForAccessLog(ServiceCtx context)
 
             return ResultCode.Success;
         }
+
+        [Command(1200)]
```
Comment (AcK77) [inline Ryujinx.HLE/HOS/Services/Fs/IFileSystemProxy.cs]:
```suggestion
        [Command(1200)] // 6.0.0+
```

File: Ryujinx.HLE/HOS/Services/Fs/IMultiCommitManager.cs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    class IMultiCommitManager : IpcService
+    {
+        private LibHac.FsService.IMultiCommitManager _baseCommitManager;
+
+        public IMultiCommitManager(LibHac.FsService.IMultiCommitManager baseCommitManager)
+        {
+            _baseCommitManager = baseCommitManager;
+        }
+
+        [Command(1)]
```
Comment (AcK77) [inline Ryujinx.HLE/HOS/Services/Fs/IMultiCommitManager.cs]:
```suggestion
        [Command(1)] // 6.0.0+
```

File: Ryujinx.HLE/HOS/Services/Fs/IMultiCommitManager.cs [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        public ResultCode Add(ServiceCtx context)
+        {
+            IFileSystem fileSystem = GetObject<IFileSystem>(context, 0);
+
+            Result result = _baseCommitManager.Add(fileSystem.GetBaseFileSystem());
+
+            return (ResultCode)result.Value;
+        }
+
+        [Command(2)]
```
Comment (AcK77) [inline Ryujinx.HLE/HOS/Services/Fs/IMultiCommitManager.cs]:
```suggestion
        [Command(2)] // 6.0.0+
```

File: NuGet.Config [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (Thog) [inline NuGet.Config]: Remove this if libhac now have a release

**Pull request merged**